pkg/ipamd/ipamd.go

```diff
@@ -1319,13 +1311,8 @@ func (c *IPAMContext) nodeIPPoolReconcile(ctx context.Context, interval time.Dur
 		}
 
 		if c.enablePodENI && metadataResult.TrunkENI != "" {
-			// Label the node that we have a trunk
-			err = c.SetNodeLabel(ctx, "vpc.amazonaws.com/has-trunk-attached", "true")
-			if err != nil {
-				podENIErrInc("askForTrunkENIIfNeeded")
-				log.Errorf("Failed to set node label for trunk. Aborting reconcile", err)
-				return
-			}
+			// Signal event to VPC RC that the node has a trunk already
```

Comment:
nit: we are not signaling anything to VPC RC here, right? Just logging locally as the annotation is already present

Reply:
Right, this was leftover after I removed the call to send event during updating some logic flow. Thanks for catching this.